esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2924,6 +2901,14 @@ bool CFileSprayEx::onDropZoneFileSearch(IEspContext &context, IEspDropZoneFileSe
         const char* dropZoneName = req.getDropZoneName();
         if (isEmptyString(dropZoneName))
             throw MakeStringException(ECLWATCH_INVALID_INPUT, "DropZone not specified.");
+        const char* dropZoneServerReq = req.getServer(); //IP or hostname
+        if (isEmptyString(dropZoneServerReq))
+            throw MakeStringException(ECLWATCH_INVALID_INPUT, "DropZone server not specified.");
+        const char* nameFilter = req.getNameFilter();
+        if (isEmptyString(nameFilter))
+            throw MakeStringException(ECLWATCH_INVALID_INPUT, "Name Filter not specified.");
+        if (streq(nameFilter, "*"))
+            throw MakeStringException(ECLWATCH_INVALID_INPUT, "Invalid Name Filter '*'");
```

Comment:
What about '**' ?  I think we should not allow a filter than contains only '*'s, not just a single '*'

Reply:
Yes, I will fix that.